blockchain/tx_list.go

```diff
@@ -281,14 +312,24 @@ func (l *txList) Add(tx *types.Transaction, priceBump uint64) (bool, *types.Tran
 	// If there's an older better transaction, abort
 	old := l.txs.Get(tx.Nonce())
 	if old != nil {
-		if tx.Type().IsCancelTransaction() {
-			logger.Trace("New tx is a cancel transaction. replace it!", "old", old.String(), "new", tx.String())
-		} else {
```

Comment:
You can see there's new version for `cancelTransaction` below.
We added the replace mechanism for Tx with extra fee than older one.
think of the situation that cannot be dealt with due to an increase in price and the user want to pay more than changed `basefee`
This code will be fixed after the governance parameters PR is merged.

Reply:
This is because the behavior before and after the hardfork must be separated. Therefore, I wrote the following at the end of the PR explanation.
`These changes need to be hardfork, i commented existing code. TODO : Need to KIP-71 hardfork.`

Reply:
Now I see. If I correctly understand it, the type of cancel transaction should be removed as well as related functionalities. Is it Correct?

Reply:
No, Cancel transaction also remained as a transaction type. It just intended to separate logic before and after hardfork.